src/useMbtaApi.js

```diff
@@ -1,3 +1,9 @@
+/*
+
+Funcs that fetch data served by the backend in the form of the Initial Request Data JSON
```

Comment:
The `export` on most of the functions in this file is an oversight on my part — the only function that _should_ be exported is `useMbtaApi`. This suggests a rewording of this comment — the file contains a React hook that provides data from the MBTA API? Where you write `Initial Request Data JSON` I think you want want to say something closer to `the __NTT_INITIAL_DATA__ JSON blob that is embedded in the initial server response`...currently it's too imprecise to be a useful signpost.